CHANGES.txt

```diff
@@ -4,14 +4,22 @@ Changelog
 2.1.0 (unreleased)
 ------------------
 
+Announcements:
+
+* Change all references to the pep8 project to say pycodestyle; #530
+
 Changes:
 
-* Added check E74x for using variables named 'l', 'O', or 'I'; #341
-* Improved performance of `compound_statements` check; #314 / #522
-* Fixed remaining references to `pep8`; #518 / #530
-* Added `noqa` support for `maximum_line_length` check; #538
-* Added check E305 for two blank lines after toplevel methods and classes; #400
+* Report E302 for blank lines before an "async def"; #556
+* Update our list of tested and supported Python versions which are 2.6 through 3.5 as well as the nightly Python build and PyPy.
```

Comment:
We should drop support for 3.2 and add a new PR to update our language to be far more specific about what versions we're testing.

Thanks @daspecster